pkg/stub/handler.go

```diff
@@ -724,7 +728,17 @@ func (h *Handler) Handle(event v1.Event) error {
 				return err
 			}
 
-			err = h.createSamples(cfg, true, registryChanged, unskippedStreams, unskippedTemplates)
+			abortForDelete, err := h.createSamples(cfg, true, registryChanged, unskippedStreams, unskippedTemplates)
+			if abortForDelete {
```

Comment:
We do this even if err != nil? Typical golang convention (and expectation) is that `err != nil` is handled first by method callers.

Reply:
/lgtm cancel

Reply:
I would call this an exception to the rule, as the delete is blowing everything away, including whatever error tracking might have happened with any error processing

Reply:
I can add a comment to that effect, assuming we can agree on this.

Reply:
added clarifying comment @adambkaplan .... PTAL